core/src/test/scala/org/apache/spark/benchmark/Benchmark.scala

```diff
@@ -162,7 +161,6 @@ private[spark] class Benchmark(
     // scalastyle:off
     println(s"  Stopped after $i iterations, ${NANOSECONDS.toMillis(runTimes.sum)} ms")
     // scalastyle:on
-    assert(runTimes.nonEmpty)
```

Comment:
Oh, actually, we could switch it to `require` instead.

Reply:
Ah, that looks better...